src/frontend/src/lib/components/ui/Badge.svelte

```diff
@@ -0,0 +1,21 @@
+<script lang="ts">
+	export let variant: 'default' | 'info' | 'error' | 'warning' | 'success' | 'outline' = 'default';
```

Comment:
There isn't a better way to type this in TypeScript in order to exactly reflects the possible types of the `variableClassNames`?

Reply:
I changed it to: `export let variant: keyof typeof variantClassNames = 'default';`